packages/postgresql/postgresql.4.4.2/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+maintainer: "Markus Mottl <markus.mottl@gmail.com>"
+authors: [
+  "Alain Frisch <alain.frisch@lexifi.com>"
+  "Markus Mottl <markus.mottl@gmail.com>"
+  "Petter Urkedal <paurkedal@gmail.com>"
+]
+license: "LGPL-2.1+ with OCaml linking exception"
+homepage: "https://mmottl.github.io/postgresql-ocaml"
+doc: "https://mmottl.github.io/postgresql-ocaml/api"
+dev-repo: "git+https://github.com/mmottl/postgresql-ocaml.git"
+bug-reports: "https://github.com/mmottl/postgresql-ocaml/issues"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" {build & >= "1.4.0"}
+  "base" {build}
+  "stdio" {build}
+  "base-bytes"
+]
+
+depexts: [
```

Comment:
I think it could be beneficial to everyone to get this depexts in a separate `conf-postgresql` package

Reply:
Thanks for the hint, I have just submitted a PR for `conf-postgresql`, which will be used in future releases of the PostgreSQL bindings: https://github.com/ocaml/opam-repository/pull/13798